### ISSUE TITLE
New Booking Enquiry notifies user but not site admin

### ISSUE THREAD

**Issue by jawnirock** [Roomify/RfA#382]
Although I get the following on the error log:
Successfully sent email to (my email address)
I do not receive the email. However, if I put a correct email on the form, the confirmation email is sent to that user (although not everytime).
Am I doing something wrong? Is there some configuration I should be changing?

**Comment by salvoscala** [comment]
Hi @jawnirock, are you talking about the general contact form or an enquiry about a single property?

**Comment by jawnirock** [comment]
Single property listing. I have the admin listed as property owner as well, but I dont receive notifications.

**Comment by salvoscala** [comment]
yeah, at the moment only the property owner receives notifications for listing enquiries. We will review the feature and notificate the roomify manager too.

Many Thanks!

**Comment by jawnirock** [comment]
Problem is, property owner is not receiving notifications either. Is there a setting section I can look around for the enquiry notifications and its components?
